### DISCUSSION TITLE
Github need to block porn on their plateform to protect children

### DISCUSSION THREAD

**Discussion by devtool15424** [community/community#86676] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi,
I find an iptv public collection. This project is really great but my child use it to watch porn and i'm really worry about it.

He is 10 years old. I'm worry about porn effect on him. And impact on his sexuality in the future. 

I would like to protect him for this. 

So this repo is here
https://github.com/iptv-org/iptv

They give an iptv collection of porn who can be se without any protection. 

https://iptv-org.github.io/iptv/categories/xxx.m3u

I would like that github block this and didn't allow to spread porn tv channel without any monitoring for children and without any age verification.

Github is not a porn website so please block all project related to porn

Have a nice day

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @devtool15424 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
